src/components/AccountContainer.js

```diff
@@ -13,49 +13,50 @@ class AccountContainer extends Component {
 
     this.state = {
       searchTerm: '',
-      transactions: [
-        {
-          id: 1,
-          posted_at: "2017-02-28 11:00:00",
-          description: "Leather Pants, Gap co.",
-          category: "Fashion",
-          amount: -20000
-        },
-        {
-          id: 2,
-          posted_at: "2017-02-29 10:30:00",
-          description: "Paycheck from Bob's Burgers",
-          category: "Income",
-          amount: 100000
-        },
-        {
-          id: 3,
-          posted_at: "2017-05-24 10:53:00",
-          description: "'Pair Programming Illuminated' by Laurie Williams and Robert Kessler",
-          category: "Book",
-          amount: 1498
-        },
-        {
-          id: 4,
-          posted_at: "2017-05-24 08:52:00",
-          description: "Medium Iced Cold Brew, Gregory's Coffee",
-          category: "Coffee",
-          amount: 365
-        }
-      ]
+      filteredTransactions: [],
+      transactions: []
     }
+    this.getTransactions = this.getTransactions.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+  getTransactions() {
+    return fetch('https://boiling-brook-94902.herokuapp.com/transactions')
+      .then(res => res.json())
+      .then(transactions => this.setState({
+        transactions: transactions,
+        filteredTransactions: transactions
+      }))
+  }
+
+  componentDidMount() {
+    this.getTransactions()
+    
   }
 
   handleChange(event) {
-    // your code here
+    event.preventDefault()
+    this.setState({
+      searchTerm: event.target.value
+    })
+    let filtered = this.state.transactions.filter(transaction => {
+      if (transaction.description.includes(event.target.value)) {
```

Comment:
you could probably shorten this by splitting these checks into a separate method